<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullscreenCameraActionWidget</class>
 <widget class="QWidget" name="FullscreenCameraActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>269</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="cameraLabel">
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="nx::vms::client::desktop::RewindForWidget" name="rewindForWidget" native="true"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="selectLayoutButton">
     <property name="text">
      <string notr="true">&lt;Select Layout&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="selectCameraButton">
     <property name="text">
      <string notr="true">&lt;Select Camera&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QnWordWrappedLabel" name="layoutWarningWidget" native="true"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="layoutLabel">
     <property name="text">
      <string>On layout</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnWordWrappedLabel</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/word_wrapped_label.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::RewindForWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/event_rules/widgets/rewind_for_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
